sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -284,6 +284,30 @@ directive:
           "$ref": "#/definitions/ConversationalAnalysisAuthoringTrainingJobState"
         }
       };
+  - where-operation: ConversationalAnalysisAuthoring_AssignDeploymentResources
+    transform: >
+      $["responses"]["200"] = {
+        "description": "dummy schema to get poller response when calling .result()",
```

Comment:
Nit: I've seen it written before we shouldn't use "dummy". Perhaps "mock" or "fake"?